src/benchmarks/micro/libraries/System.Runtime/Perf.DateTime.cs

```diff
@@ -13,7 +13,9 @@ public class Perf_DateTime
     {
         DateTime date1 = new DateTime(1996, 6, 3, 22, 15, 0);
         DateTime date2 = new DateTime(1996, 12, 6, 13, 2, 0);
-        
+
+        readonly DateTime utcNow = DateTime.UtcNow;
```

Comment:
Please let's use a fixed date. With all benchmarks we use deterministic inputs. Even though it's unlikely it matters here, I'm reminded of https://github.com/dotnet/runtime/issues/67932..

Reply:
Changed to fixed datetime.
There is still branching in `.Month` implementation. But it is insignificant.